azurerm/internal/services/loganalytics/tests/log_analytics_cluster_resource_test.go

```diff
@@ -0,0 +1,282 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/loganalytics/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMLogAnalyticsCluster_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_log_analytics_cluster", "test")
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMLogAnalyticsClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMLogAnalyticsCluster_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMLogAnalyticsClusterExists(data.ResourceName),
+				),
+			},
+			data.ImportStep("size_gb"), // not returned by the API
```

Comment:
This feels like a bug, it should be present in the Sku?

Reply:
It is in the `Sku` I collapsed the `Sku` as it has only one possible value. The only value that is possible for the end user to change is `capacity` value in the `Sku` so I pulled that out to a top level attribute and by doing so that causes the `ImportStep` to fail because the state and `GET` do not match.